wrapping/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(${ITK_WRAP_float} AND NOT ${ITK_WRAP_complex_float})
 endif()
 
 itk_wrap_module(Ultrasound)
-set(WRAPPER_LIBRARY_GROUPS
+Set(WRAPPER_LIBRARY_GROUPS
```

Comment:
Capitalized command? Was this accidental or purposeful?

Reply:
Good catch ... accidental!